tests/compile/test_fusion_attn.py

```diff
@@ -407,9 +414,13 @@ def test_attention_quant_pattern(num_qo_heads: int, num_kv_heads: int,
                                     vllm_config=vllm_config_unfused)
         model_unfused = model_unfused.to(device)
 
-        forward_ctx = get_forward_context()
-        forward_ctx.attn_metadata = model_unfused.build_attn_metadata(
-            batch_size, use_hnd=split_attention)
+        # TODO(boyuan): the attn_metadata with quantization does not
+        # work on my server. So skip for inductor graph partition
+        # test for now.
```

Comment:
fwiw I can't run these tests on my B200 machine either

Reply:
The test works on CUDA Driver 580 but not on 575